make_profiler/report/script.js

```diff
@@ -0,0 +1,148 @@
+function filterTarget() {
+    var input, filter, table, tr, td, i, txtValue;
+    input = document.getElementById("myInput");
+
+    filter = input.value.toUpperCase().split(" ").join("_");
+    table = document.getElementById("statusTable");
+    tr = table.getElementsByTagName("tr");
+
+    if (filter.startsWith('"') && filter.endsWith('"')) {
+        filter = filter.substring(1, filter.length - 1)
+        for (i = 0; i < tr.length; i++) {
+            td = tr[i].getElementsByTagName("td")[0];
+            if (td) {
+                txtValue = td.childNodes[0].textContent || td.innerText;
+                if (txtValue.toUpperCase() === filter) {
+                    tr[i].style.display = "";
+                } else {
+                    tr[i].style.display = "none";
+                }
+            }
+        }
+    } else {
+        for (i = 0; i < tr.length; i++) {
+            td = tr[i].getElementsByTagName("td")[0];
+            if (td) {
+                txtValue = td.textContent || td.innerText;
+                if (txtValue.toUpperCase().indexOf(filter) > -1) {
+                    tr[i].style.display = "";
+                } else {
+                    tr[i].style.display = "none";
+                }
+            }
+        }
+    }
+}
+
+//refresh page every 5 minutes
+setTimeout(function () {
+    window.location.reload(1);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Location/reload

reload doesn't have a parameter, isn't it?